docs/glossary.md

```diff
@@ -69,7 +69,7 @@ You can't onboard a GSIB device to SEED.
 
 It is a Secured Email (SE) GSIB device. This GSIB device is assigned to public officers who handle sensitive and secret information.
 
-If you are using a SE GSIB device, you will be using your PS-Card to authenticate. You will not be able to access internet using this SE-GSIB device.
+If you are using a SE GSIB device, you need to use your PS-Card to authenticate. You will not be able to access internet using this SE-GSIB device.
```

Comment:
Can we use "and you will not be about to access the Internet using this device"?

the Internet is capital I -> unless i am wrong on this :S

Reply:
confirmed with Microsoft style guide https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/i/internet-intranet-extranet. Apparently, no need to capitalise :-).